projects/bent/index.js

```diff
@@ -22,6 +22,7 @@ async function tvl(api) {
 module.exports = {
   methodology: `TVL:BENT allows users to stake their curve LP tokens. For each supported curve pool LP token: Find the total supply of the LP token, Find the balance of LP staked in bent, Find the curve pool whose liquidity it represents, Enumerate the addresses of each token that makes up the pool, Get the balance of each token from 4, Use 1 & 2 to work out the LP share staked in bent, Multiply the token balances from 5 by the bent share to get the bent balances. Pool2 and staking: Pool2 and staking are fairly standard. Pool2 calculates fraction of LP staked as a share of the sushi LP and multiplies by the sushi LP coin balances. Staking simply takes the balance of the staking contract for BENT.`,
   ethereum: {
+    doublecounted: true,
```

Comment:
protocol is categorized as Yield, all yield protocols are by nature doublecounted and  thus excluded from default tvl